### PR TITLE
Compile the library

### PR DESCRIPTION
**Pull request by jspricke**

### CODE REVIEW

**Comment by jspricke** [comment]
Same as #18 (just untested ;) ).

File: libreflexxestype2/CMakeLists.txt [hunk 1]
```diff
@@ -1,55 +1,53 @@
...
+src/TypeIIRML/TypeIIRMLVelocityIsPhaseSynchronizationPossible.cpp
+src/TypeIIRML/TypeIIRMLVelocityMethods.cpp
+src/TypeIIRML/TypeIIRMLVelocitySetupPhaseSyncSelectionVector.cpp
+)
+
+install(
+  TARGETS
+    reflexxes
+  LIBRARY DESTINATION
+    lib
```
Comment (AndyZe) [inline libreflexxestype2/CMakeLists.txt:43]: Parse error.  Function missing ending ")".  End of file reached.

Comment (AndyZe) [reply libreflexxestype2/CMakeLists.txt:43]: It builds if I just add that parantheses. That's good enough for me, considering this package doesn't get much use. Thanks for the PR.

Comment (AndyZe) [inline libreflexxestype2/CMakeLists.txt:43]:
```suggestion
    lib
)
```

**Pull request merged**